content/articles/dnsimple-services.md

```diff
@@ -4,54 +4,76 @@ excerpt: "DNSimple provides essential services for every Internet-connected syst
+We bill domain registrations and transfers on a yearly basis. Pricing varies depending on the TLD (Top-Level Domain). For more information, view our [TLD price list](https://dnsimple.com/tlds).
+
+We also support [domain transfers](/articles/domain-transfer/) to and from DNSimple and offer domain [auto-renewal](/articles/domain-auto-renewal/).
+
+Want to use our DNS hosting, but not registration? You have the option to use another registrar and [change your name servers to our name servers](/articles/delegating-dnsimple-hosted/). DNSimple customers aren't required to use our domain registration services to take advantage of our hosted DNS.
 
 ## Powerful automation API
 
-With the DNSimple REST API, you can register domain names and administrate them directly from your own scripts. We have a [repository filled with API examples](https://github.com/dnsimple/dnsimple-api-examples), so you can get up and running. Some things you can do with our API include:
+With DNSimple's REST API, you can register domain names and administrate them directly from your own scripts. We have a repository [filled with API examples](https://github.com/dnsimple/dnsimple-api-examples), so you can get up and running.
+
+Some things you can do with our API include:
 
 - Check domain availability.
 - Register and renew domains.
 - Transfer domains.
 - Buy SSL certificates.
 - List and Manage Records.
 - Register to events using [webhooks](https://dnsimple.com/webhooks) to receive notification of changes.
+- [Query DNS analytics data](https://developer.dnsimple.com/v2/dns-analytics/).
 
-Read up on our [API features and abilities](https://dnsimple.com/api).
+Read more about our [API features and abilities](https://dnsimple.com/api).
 
-## One Click DNS Services
+## One-click DNS services
 
-One Click Services are ready-to-use DNS snippets. Applying a service creates all the DNS configurations necessary to configure your email and serve up your webpage.
+One-click services create the DNS configuration needed for web pages, email, e-commerce, and more — add all the necessary DNS records for any available service with a single click.
 
 We offer:
 
-- Hosting services, like Surge, WordPress, Webflow, and more.
-- Productivity services, including Google's G-Suite, Office 365, and KickoffLabs.
-- Infrastructure services like Heroku.
+- Hosting services, like Weebly, WordPress, Webflow, and more.
+- Productivity services, including Google Workspace, Office 365, and KickoffLabs.
+- Infrastructure services like Heroku and Netlify.
 - Email services, including Rackspace, Mailgun, and Fastmail.
-- Ecommerce services, like Shopify and Squarespace.
+- E-commerce services, like Shopify and Squarespace.
+
+For a complete list of our One-click services and how to add them, (view this article)[/articles/services/].
```

Comment:
should we also link to the integrations page? https://dnsimple.com/benefits/integrations

Reply:
I had thought about it, but I'm not sure it adds anything unless it's useful for SEO